vulnerable.cpp

```
@@ -1 +1,108 @@
+#include <thread>
+#include <cstdlib>
+#include <unistd.h>
 
+using namespace std;
+
+int globalCounter = 0;
+
+class VulnerableServer {
+public:
+    VulnerableServer() {
+        adminUser = "admin";
```

Cppcheck (2.10-2)

[performance] Variable 'adminUser' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)